.circleci/config.yml

```diff
@@ -75,40 +52,6 @@ commands:
           keys:
             - cimg-go-pkg-mod-{{ arch }}-{{ checksum "go.sum" }}
 
-  publish_docker_images:
-    parameters:
-      repo:
-        type: string
-      tag:
-        type: string
-
-    steps:
-      - run:
-          name: Build image
-          command: |
-            make docker-otelcol
-            docker tag otelcol:latest otel/<< parameters.repo >>:<< parameters.tag >>
-            docker tag otelcol:latest otel/<< parameters.repo >>:latest
-      - run:
-          name: Login to Docker Hub
-          command: docker login -u $DOCKER_HUB_USERNAME -p $DOCKER_HUB_PASSWORD
-      - run:
-          name: Push image
-          command: |
-            docker push otel/<< parameters.repo >>:<< parameters.tag >>
-            docker push otel/<< parameters.repo >>:latest
-
-  github_issue_generator:
```

Comment:
This step appears unrelated to publishing releases.

Reply:
Nobody uses it :D was not a step but a command so if nobody is calling it it does not get executed